utils/writeMotor flowchart/code/code.txt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2022 Carry Assistant Trolley (C.A.T) Technologies
+ *  
+ * Adwait P. Naik, February 27, 2022.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02111, USA.
+ */
+
+writeMotor;
+Initialize the variables to store Left/Right Motor's analog output;
+branch(left_motor)
```

Comment:
Does this write motor flow get involved with the compute angle flow in anyway? 
Just wanted to understand if the analog output has any relation with RSSI converted distance/compute angle stuff or it is completely independent or dependent on something else?

I think the main flow diagram you made might depict this..